src/utils.js

```diff
@@ -41,6 +41,10 @@ let trimPath = (path) => {
     return path?.replace(/^\//, '');
 }
 
+let clearCache = () => {
+    fs.removeSync('temp');
```

Comment:
Try to make it more generic by passing the folder name that is intended to be deleted.
Let the function accept the `dirName` in params.